src/data/polynomial/coeff.lean

```diff
@@ -174,7 +174,7 @@ begin
   rw [← f.sum_single] {occs := occurrences.pos [1]},
   refine sum_mem _ (λ i hi, _),
   change monomial i _ ∈ span _ _,
-  rw [← C_mul_X_pow_eq_monomial, ← X_pow_mul],
+  rw [← C_mul_X_pow_eq_monomial, ← X_pow_mul, ← smul_eq_mul],
```

Comment:
Did this one fail in the same way as https://github.com/leanprover-community/mathlib/pull/6590/files#r593184827?

Reply:
I am trying to find the build error, but I could not.  In any case, it was complaining since again there was a mul/smul mismatch.  The more I think about this, the more I think that mul should not exist and everything should be a smul!  :wink:

Reply:
Found it!

```lean
Run leanpkg configure
configuring mathlib 0.1
Error: /home/runner/work/mathlib/mathlib/src/data/polynomial/coeff.lean:178:34:
invalid constructor ⟨...⟩, expected type is not an inductive type
  ?m_2 ∈ ?m_3
Additional information:
/home/runner/work/mathlib/mathlib/src/data/polynomial/coeff.lean:178:22: context: switched to simple application elaboration procedure because failed to use expected type to elaborate it, error message
  type mismatch, term
    subset_span ?m_8
  has type
    ?m_2 ∈ ↑(span ?m_3 ?m_7)
  but is expected to have type
    ?m_6 ∈ ?m_7
/home/runner/work/mathlib/mathlib/src/data/polynomial/coeff.lean:178:8: context: switched to simple application elaboration procedure because failed to use expected type to elaborate it, error message
  type mismatch, term
    ?m_6.smul_mem ?m_8 ?m_9
  has type
    ?m_6 • ?m_7 ∈ ?m_8
  but is expected to have type
    X ^ i * ⇑C (⇑f i) ∈ span (polynomial R) {g : polynomial R | ∃ (i : ℕ), g = ⇑C (f.coeff i)}
state:
R : Type u,
_inst_1 : semiring R,
f : polynomial R,
i : ℕ,
hi : i ∈ f.support
⊢ X ^ i * ⇑C (⇑f i) ∈ span (polynomial R) {g : polynomial R | ∃ (i : ℕ), g = ⇑C (f.coeff i)}
Error: Process completed with exit code 1.
```